src/sections/blog-post.liquid

```diff
@@ -0,0 +1,147 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is available when you add "Blog Posts" section on the Theme Editor.
+
+  Theme Store required settings
+  - Blog (which blog category to show in your store)
+  - Number of posts to show
+
+  Theme Store optional settings
+  - Show author
+  - Show date
+{%- endcomment -%}
+<section class="blog-post">
+  {% if section.settings.title != blank %}
+    <h2 class="blog-post__title">{{ section.settings.title | escape }}</h2>
+  {% endif %}
+
+  {%- comment -%}
+    This refer from blog section when you add the section from Theme Editor
+  {%- endcomment -%}
```

Comment:
Do we need this comment? We have other instances where we're doing something similar for a selected `collection`, and not including a comment.